src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -99,7 +99,7 @@ def restoresssdconf():
                          'install_nslcd',
                          'create_user_with_cn',
                          'netgroups')
-@pytest.mark.tier1_3
+@pytest.mark.tier2
```

Comment:
Add this marker for test `test_bz1368467` only, rest of the tests can continue to run in tier1_3